tests/BunqSdkTestBase.php

```diff
@@ -3,61 +3,246 @@
+     */
+    protected $secondMonetaryAccountBank;
 
     /**
-     * Device description for PHP unit tests
+     * @var CashRegister
      */
-    const DEVICE_DESCRIPTION = 'PHP unit tests';
+    protected $cashRegister;
 
     /**
      */
     public static function setUpBeforeClass()
     {
-        static::ensureApiContextValid();
+        static::createApiContext();
+        BunqContext::loadApiContext(
+            ApiContext::restore(self::FILE_PATH_CONTEXT_CONFIG)
+        );
+    }
+
+    /**
+     */
+    protected static function createApiContext()
+    {
+        InstallationUtil::automaticInstall(
+            BunqEnumApiEnvironmentType::SANDBOX(),
+            self::FILE_PATH_CONTEXT_CONFIG
+        );
+    }
+
+    /**
+     */
+    protected function setUp()
+    {
+        $this->setSecondMonetaryAccountBank();
+    }
+
+    /**
+     */
+    private function setCashRegister()
+    {
+        $attachmentUuid = AttachmentPublic::create(
+            FileUtil::getFileContents(__DIR__ . self::FILE_PATH_AVATAR),
+            [
+                ApiClient::HEADER_CONTENT_TYPE => $this->getAttachmentContentType(),
+                ApiClient::HEADER_ATTACHMENT_DESCRIPTION => $this->getAttachmentDescription(),
+            ]
+        );
+        $avatarUuid = Avatar::create($attachmentUuid->getValue());
+        $cashId = CashRegister::create(
+            self::CASH_REGISTER_NAME,
+            self::CASH_REGISTER_STATUS,
+            $avatarUuid->getValue()
+        );
+
+        $this->cashRegister = CashRegister::get($cashId->getValue());
+    }
+
+    /**
+     * @return string
+     */
+    protected function getAttachmentContentType(): string
+    {
+        return self::ATTACHMENT_CONTENT_TYPE;
     }
 
     /**
-     * Ensures the API context is still valid.
+     * @return string
      */
-    protected static function ensureApiContextValid()
+    protected function getAttachmentDescription(): string
     {
-        try {
-            $apiContext = ApiContext::restore(static::FILENAME_CONTEXT_CONFIG);
-            $apiContext->ensureSessionActive();
-        } catch (BunqException $exception) {
-            $apiContext = self::createApiContext();
+        return self::ATTACHMENT_DESCRIPTION;
+    }
+
+    /**
+     */
+    private function setSecondMonetaryAccountBank()
+    {
+        $createdId = MonetaryAccountBank::create(
+            self::MOMENTARY_ACCOUNT_CURRENCY,
+            self::MONETARY_ACCOUNT_DESCRIPTION
+        );
+
+        $this->secondMonetaryAccountBank = MonetaryAccountBank::get($createdId->getValue())->getValue();
+    }
+
+    /**
+     * @return Pointer
+     *
+     * @throws BunqException
+     */
+    protected function getSecondMonetaryAccountAlias(): Pointer
+    {
+        $allAlias = $this->secondMonetaryAccountBank->getAlias();
+
+        foreach ($allAlias as $alias) {
+            if ($alias->getType() === self::POINTER_TYPE_IBAN) {
+                return $alias;
+            }
```

Comment:
Explicit else

Reply:
In open-source project explicit else is not needed. Only in bunq internal projects.